sources/packages/backend/apps/api/src/route-controllers/student-appeal/_tests_/e2e/student-appeal.students.controller.submitStudentAppeal.e2e-spec.ts

```diff
@@ -0,0 +1,291 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource, IsNull } from "typeorm";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  FakeStudentUsersTypes,
+  getStudentToken,
+  mockJWTUserInfo,
+  resetMockJWTUserInfo,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeStudentAppeal,
+  createFakeStudentAppealRequest,
+  getProviderInstanceForModule,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { TestingModule } from "@nestjs/testing";
+import { AppStudentsModule } from "../../../../app.students.module";
+import { FormService } from "../../../../services";
+import {
+  NotificationMessageType,
+  StudentAppealActionType,
+  StudentAppealStatus,
+} from "@sims/sims-db";
+import MockDate from "mockdate";
+import {
+  getDateOnlyFormat,
+  getPSTPDTDateTime,
+} from "@sims/utilities/date-utils";
+import { STUDENT_HAS_PENDING_APPEAL } from "../../../../constants";
+
+describe("StudentAppealStudentsController(e2e)-submitStudentAppeal", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  let appModule: TestingModule;
+  let db: E2EDataSources;
+  let formService: FormService;
+  const MODIFIED_INDEPENDENT_APPEAL = "modifiedindependentappeal";
+  const ENDPOINT = "/students/appeal";
+  const MINISTRY_EMAIL_ADDRESS = "dummy@some.domain";
+  const MODIFIED_INDEPENDENT_FORM_DATA = {
+    actions: [StudentAppealActionType.UpdateModifiedIndependent],
+    isModifiedIndependent: "no",
+  };
+  const VALID_PAYLOAD = {
+    formName: MODIFIED_INDEPENDENT_APPEAL,
+    formData: MODIFIED_INDEPENDENT_FORM_DATA,
+    files: [],
+  };
+
+  beforeAll(async () => {
+    const { nestApplication, module, dataSource } =
+      await createTestingAppModule();
+    app = nestApplication;
+    appDataSource = dataSource;
+    appModule = module;
+    db = createE2EDataSources(dataSource);
+    // Update fake email contact to send ministry email.
+    await db.notificationMessage.update(
+      {
+        id: NotificationMessageType.StudentSubmittedChangeRequestNotification,
+      },
+      { emailContacts: [MINISTRY_EMAIL_ADDRESS] },
+    );
```

Comment:
The ministry email address should be something shared across the API tests, otherwise it could conflict with other tests.